src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Model/Column.cs

```diff
@@ -434,6 +434,30 @@ public override void Set(SqlDataRecord record, int ordinal, byte? value)
         }
     }
 
+    public class TimestampColumn : Column<byte[]>
+    {
+        public TimestampColumn(string name)
+            : base(name, SqlDbType.Timestamp, true)
+        {
+        }
+
+        public override byte[] Read(SqlDataReader reader, int ordinal)
+        {
+            // The row version storage size is 8 bytes.
```

Comment:
nit: might want to indicate that this is synonymous with row version in the class documentation.

Reply:
I agree! Added in d6cbf1f.